slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/SignedAttestationsDao.java

```diff
@@ -118,4 +118,20 @@ public void insertAttestation(final Handle handle, final SignedAttestation signe
                 + "FROM signed_attestations WHERE validator_id = ?")
         .bind(0, validatorId).mapToBean(SignedAttestation.class).stream();
   }
+
+  public void deleteAttestationsBelowEpoch(
+      final Handle handle, final int validatorId, final UInt64 epoch) {
+    handle.execute(
```

Comment:
feels like a really dangerous function to have openly available - i.e. what if epoch is above the watermark?

Reply:
same goes for blocks

Reply:
Going to update this to only delete data below watermark